src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -669,6 +671,23 @@ bool mongoLocationCapable(void)
 
 
 
+/* ***************************************************************************
+*
+* mongoExpirationCapable -
+*/
+bool mongoExpirationCapable(void)
+{
+  int mayor;
+  int minor;
+
+  /* TTL (Time To Live) indexes was introduced in MongoDB 2.2 */
+  mongoVersionGet(&mayor, &minor);
+
+  return ((mayor == 2) && (minor >= 2)) || (mayor > 2);
```

Comment:
Not sure about this...

Although TTL index were introduced in MongoDB 2.2 according to this piece of documentation https://docs.mongodb.com/v2.2/tutorial/expire-data, the usage of 0 as value of expireAfterSeconds in order to expire at a specific date is not documented there. However https://docs.mongodb.com/v2.4/tutorial/expire-data uses that 0.

Maybe is more correct to check for 2.4. And tune the comment in the following way:

```
/* TTL (Time To Live) indexes was introduced in MongoDB 2.2, although the expireAfterSeconds: 0 usage is not shown in documentation until 2.4 */
```

I may be wrong... other pieces of documentation could say otherwise ;). In that case, please cite them.

Reply:
Fixed in 5a4703d